### DISCUSSION TITLE
tech/2021/03/typo-keyboard

### DISCUSSION THREAD

**Discussion by giscus** [shikhalev/shikhalev.github.io#15] [General]
# 2021/03/typo-keyboard

Настройка клавиатуры в Linux для ввода различных символов, таких как, например, длинное тире, кавычки-ёлочки, знак копирайта и так далее. В редакции 2021 года.

https://shikhalev.github.io/tech/2021/03/typo-keyboard.html

**Comment by nonameformr** [comment]
Благодарю за статью.
Подскажи, не пойму, как назначить на 3м уровне те самые символы, которые нужны?
Например, я печатаю на русской раскладке и мне необходимо ввести # без переключения на английскую, я могу с помощью 3го ряда это сделать?

**Reply by shikhalev** [reply]
Сделать это можно, и собственно в части «Кастомизируем раскладку» я это и описываю. Другое дело, что подключать эту кастомизацию несколько муторно (хотя в Wayland должно быть попроще, но у меня с ним другие проблемы, так что не пробовал).
